python/ctsm/modify_input_files/fsurdat_modifier.py

```diff
@@ -429,10 +434,26 @@ def read_cfg_option_control(
         logger.info("dom_pft option is on and = %s", str(dom_pft))
     else:
         logger.info("dom_pft option is off")
+    evenly_split_cropland = get_config_value(
+        config=config,
+        section=section,
+        item="evenly_split_cropland",
+        file_path=cfg_path,
+        convert_to_type=bool,
+    )
+    if evenly_split_cropland and dom_pft and dom_pft > int(max(modify_fsurdat.file.natpft.values)):
```

Comment:
Does `and dom_pft` need to appear before the second `and dom_pft`? Does it ensure that dom_pft is not None? Or is it possibly unnecessary?

Reply:
Other than this, I, too, have approved the PR.

Reply:
Thanks for pointing this out @slevis-lmwg. As is it looks confusing. But, the second "and" actually goes with

(dom_pft > int(...))

So @samsrabin I just suggest you add parenthesis around that last expression to make it clearer. You might also add a comment about it because the logic is a bit obtuse. It's checking that evenly_split_cropland is set, as well as dom_pft (to something not 0), and that dom_pft is greater than the natural pft values, meaning that it's some sort of crop type.

Reply:
Good points, both. The `and dom_pft` was indeed there to ensure `dom_pft is not None`, but I see how it's confusing.

I've cleaned this up in 0f11277; what do you think?

Reply:
Your new commit makes it clear to me. Is that what it looks like after running through black? Also speaking of, we should make sure this is all run through black and pylint...

I think the new form doesn't require a comment either as it's clear enough in that form...

Reply:
Yep, that's the formatting `black` did for it. The automated `black` check here is passing, but I hadn't run `pylint`. I just did and resolved its one complaint in [73895d6](https://github.com/ESCOMP/CTSM/pull/2081/commits/73895d6353b512f31487700459d47f932f94b8a9).

Reply:
Awesome. Looks good I'll resolve this conversation.